docs/team/chengjiyuqing.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: page
+title: Cheng Jiyuqing's Project Portfolio Page
+---
+
+## Project: AddressBook Level 3
+
+Medmoriser is a desktop app to help medical students organise, memorise and revise their content. The user interacts with it using a CLI, and it has a GUI created with JavaFX and is written in Java.
+
+Given below are my contributions to the project.
+
+* **New Feature**: Added the ability to undo/redo previous commands.
```

Comment:
I think you forgot to edit this part since we don't have an undo/redo command

Reply:
okay changed!